client/src/components/edit-client-page/index.jsx

```diff
@@ -0,0 +1,353 @@
+import * as React from "react";
```

Comment:
This is what the edit page looks like on my end at the moment. Please change it to look more like the figma design (specifically assuring the elements for editing the fields span most of the page as they do in the designs and that the icons are aligned on the right hand side of the table)
<img width="1744" alt="Screenshot 2024-01-09 at 7 29 48 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/a529d30f-75e6-4938-a4a3-6aeba3cb21fa">

Reply:
Great comment. Especially since we have different devs working on each page we want it to be uniform across the app